finders.go

```diff
@@ -66,24 +66,29 @@ func (c *Connection) First(model interface{}) error {
 //
 //	q.Where("name = ?", "mark").First(&User{})
 func (q *Query) First(model interface{}) error {
+	var m *Model
 	err := q.Connection.timeFunc("First", func() error {
 		q.Limit(1)
-		m := NewModel(model, q.Connection.Context())
+		m = NewModel(model, q.Connection.Context())
 		if err := q.Connection.Dialect.SelectOne(q.Connection, m, *q); err != nil {
 			return err
 		}
-		return m.afterFind(q.Connection)
+		return m.afterFind(q.Connection, false)
 	})
 
 	if err != nil {
 		return err
 	}
 
 	if q.eager {
-		err = q.eagerAssociations(model)
+		err := q.eagerAssociations(model)
```

Comment:
Just curious if there is a specific reason for adding ':' here. If there is a specific reason, an inline comment could be useful.

Reply:
It's mostly a coding habit on my end, also fine either way